### DISCUSSION TITLE
Cannot create OpenGL context

### DISCUSSION THREAD

**Discussion by mutallibshaik** [carla-simulator/carla#5137] [Q&A]
![image](https://user-images.githubusercontent.com/22893322/152483662-9cbb85a6-9876-4ce7-8ab2-2c871502d18a.png)

Please help

**Comment by MertBalkan** [comment]
This error can occur due to the following causes:

*The local machine does not have the necessary graphic drivers installed.
*The graphics card does not support OpenGL version 2.1 or the higher versions.

You may need  to update your Graphics Card. [You can check here](https://support.microsoft.com/en-us/windows/update-drivers-in-windows-ec62f46c-ff14-c91d-eead-d7126dc1f7b6)

Also you may need to delete and reinstall it.
